src/rate_limiter/src/lib.rs

```diff
@@ -569,30 +529,13 @@ pub(crate) mod tests {
         assert!(*tb.get_last_update() >= before);
         let after = Instant::now();
         assert!(*tb.get_last_update() <= after);
-        assert_eq!(tb.get_processed_capacity(), 1);
-        assert_eq!(tb.get_processed_refill_time(), 1_000_000);
 
         // Verify invalid bucket configurations result in `None`.
         assert!(TokenBucket::new(0, 1234, 1000).is_none());
         assert!(TokenBucket::new(100, 1234, 0).is_none());
         assert!(TokenBucket::new(0, 1234, 0).is_none());
     }
 
-    #[test]
-    fn test_token_bucket_preprocess() {
-        let tb = TokenBucket::new(1000, 0, 1000).unwrap();
-        assert_eq!(tb.get_processed_capacity(), 1);
-        assert_eq!(tb.get_processed_refill_time(), NANOSEC_IN_ONE_MILLISEC);
-
-        let thousand = 1000;
-        let tb = TokenBucket::new(3 * 7 * 11 * 19 * thousand, 0, 7 * 11 * 13 * 17).unwrap();
-        assert_eq!(tb.get_processed_capacity(), 3 * 19);
-        assert_eq!(
-            tb.get_processed_refill_time(),
-            13 * 17 * (NANOSEC_IN_ONE_MILLISEC / thousand)
-        );
-    }
-
```

Comment:
It would be nice to add unit tests that make sure we don't have the issue any more (or the issue that @roypat) mentions in his review.

Reply:
I've added a test: https://github.com/firecracker-microvm/firecracker/pull/3370#discussion_r1072184741